content/docs/start/data-management/index.md

```diff
@@ -1,294 +1,17 @@
 ---
-title: 'Get Started: Data and Model Versioning'
-description: 'Get started with data and model versioning in DVC. Learn how to
-use a regular Git workflow for datasets and ML models, without storing large
-files in Git.'
+title: 'Get Started: Data Management'
+description: 'Get started with data management in DVC, including versioning
+access, pipelines, as well as metrics, parameters, and plots.'
 ---
 
-# Get Started: Data Versioning
+# Get Started: Data Management
 
-How cool would it be to make Git handle arbitrarily large files and directories
-with the same performance it has with small code files? Imagine cloning a
-repository and seeing data files and machine learning models in the workspace.
-Or switching to a different version of a 100Gb file in less than a second with a
-`git checkout`. Think "Git for data".
+Let's look at DVC's features from the perspective of data and machine learning
+model management. This includes automatic caching; versioning on top of Git
+(without storing in the Git repo); sharing, exploring, and accessing remotely,
+among other tasks.
```

Comment:
OK so now the content in this page is completely new, just the title for the trail is kept...